bigquery/cloud-client/authorized_view_tutorial_test.py

```diff
@@ -14,6 +14,7 @@
 
 from google.cloud import bigquery
 import pytest
+import uuid
```

Comment:
Can you do the following?

```python
import uuid

from google.cloud import bigquery
import pytest

import authorized_view_tutorial
```

Usually imports should be 1) standard libs 2) 3rd party libs and 3) package local libs, separated by an empty line.